src/date.spec.js

```diff
@@ -393,61 +393,66 @@ describe('uiDateFormat', function() {
   beforeEach(module('ui.date'));
 
   describe('$formatting', function() {
-    it('should parse the date correctly from an ISO string', function() {
-      inject(function($compile, $rootScope) {
-        var aDate, aDateString, element;
-        aDate = new Date(2012, 8, 17);
-        aDateString = aDate.toISOString();
-
-        element = $compile('<input ui-date-format ng-model="x"/>')($rootScope);
-        $rootScope.x = aDateString;
-        $rootScope.$digest();
-
-        // Check that the model has not been altered
-        expect($rootScope.x).toEqual(aDateString);
-        // Check that the viewValue has been parsed correctly
-        expect(element.controller('ngModel').$viewValue).toEqual(aDate);
-      });
-    });
+    //TODO(consider) logic change - now we convert all inputs to Date object, if user wants store ISO string or other
```

Comment:
I'm concerned with this breaking change, many people seem to be passing in ISO date string in the model to start with.  I had thought we were converting to a date obj already but it's been awhile since reviewing.